app/src/main/res/values-fr/strings.xml

```diff
@@ -199,13 +199,13 @@
     <string name="swap_kiss_button_with_menu">Afficher le bouton KISS sur la droite</string>
     <string name="notification_support">Afficher un rond sur les applis avec des notifications</string>
     <string name="notification_dialog">Cochez KISS dans l\'écran suivant pour activer.</string>
-    <string name="cant_pin_shortcut">Impossible d\'ajouter le raccourci. KISS est-il bien votre lanceur par défaut \?</string>
+    <string name="cant_pin_shortcut">Impossible d\'ajouter le raccourci. KISS est-il bien votre lanceur par défaut \?</string>
     <string name="search_results_options">Résultats de recherche</string>
     <string name="search_provider_toggle">Liens des sources de recherche web</string>
-    <string name="copy_confirmation">Copié dans le presse-papier</string>
+    <string name="copy_confirmation">Copié dans le presse-papiers</string>
```

Comment:
See the French Academy's dictionary: https://www.dictionnaire-academie.fr/article/A9P4183